services/QuillLMS/app/controllers/api/v1/concept_feedback_controller.rb

```diff
@@ -2,6 +2,7 @@ class Api::V1::ConceptFeedbackController < Api::ApiController
   before_action :activity_type
   before_action :concept_feedback_by_uid, except: [:index, :create, :update]
 
+  # TODO: Cache this once routes are fixed
```

Comment:
The goal in this PR is to change the route from `api/v1/concept_feedback.json?activity_type=grammar` to `api/v1/activity_type/grammar/concept_feedback.json` so that action_cache can be used here.